packages/terra-menu/src/MenuItem.jsx

```diff
@@ -311,7 +356,14 @@ class MenuItem extends React.Component {
     return (
       <ThemeContext.Consumer>
         { theme => (
-          <li {...attributes} className={classNames(itemClassNames, cx(theme.className))} ref={this.setItemNode} role={role} aria-checked={markAsToggled}>
+          <li
+            {...attributes}
+            className={classNames(itemClassNames, cx(theme.className))}
+            ref={this.setItemNode}
+            role={role}
+            aria-checked={markAsToggled}
+            data-menuitem="true"
```

Comment:
Can we make this data attribute to more specific to active elements instead of adding it for all and making selector complicated while retrieving the item 
```js
const attributes = { ...customProps };
    attributes['data-terra-menu-interactive-item'] = isDisabled ? true : false;
```

Reply:
https://github.com/cerner/terra-framework/pull/1626/commits/f1ba5a1a0095913480262bfaa914fb730ebf7c95